setup.py

```diff
@@ -54,13 +59,16 @@
     version=__version__,
     description=('The ADAL for Python library makes it easy for python ' +
                  'application to authenticate to Azure Active Directory ' +
-                 '(AAD) in order to access AAD protected web resources.'),
+                 '(AAD) in order to access AAD protected web resources. ' +
+                 '(It is now SUPERSEDED by MSAL Python.)'),
```

Comment:
>It is now SUPERSEDED by MSAL Python.)' [](start = 19, length = 38)

Suggestion: Put this first?

Reply:
Good point. Added something similar. Please review again.